pkg/api/types.go

```diff
@@ -1205,6 +1221,13 @@ type KeyToPath struct {
 	Mode *int32
 }
 
+// Local represents a directory with specific topology contraints
+type LocalVolumeSource struct {
+	// The full path to the volume on the node
+	// For alpha, this path must be a file-based volume
+	Path string
```

Comment:
I'm interested in this discussion too.

Above proposal looks good to me but I have a question. 
Is the volumeType parameter mandatory required?
Or user can skip the parameter then the default value is set to "file".